src/cf-icons/src/cf-icons.less

```diff
@@ -44,24 +44,24 @@
 //
 // For example you can't do this:
 // .link-with-auto-icon {
-//   .@{cf-icon-prefix}();
+//   .@{cf-icon-prefix}(  );
 // }
 //
 // But you can do this:
 // .link-with-auto-icon {
-//   .@cf-icon();
+//   .@cf-icon(  );
 // }
 .cf-icon,
 .@{cf-icon-prefix} {
-  font-family: 'CFPB Minicons';
-  display: inline-block;
-  font-style: normal;
-  font-weight: normal;
-  line-height: 1;
-  // TODO: Determine whether this can be safely removed and remove it.
-  //       Font smooth/smoothing is non-standard:
-  //       https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
-  -webkit-font-smoothing: antialiased;
+    font-family: 'CFPB Minicons';
+    display: inline-block;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 1;
+    // TODO: Determine whether this can be safely removed and remove it.
+    //       Font smooth/smoothing is non-standard:
+    //       https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
```

Comment:
I'm probably also in favor of removing, but, as @KimberlyMunoz said, we should definitely do some testing to know exactly who/what it will affect. I don't want to just do it on the intuition that it'll probably be fine.